src/lib/patch/index.ts

```diff
@@ -1,11 +1,35 @@
+import { version } from '../../../package.json'
 import type { Params } from '$lib/audio'
 
+export enum Visibility {
+  public,
+  private,
+}
```

Comment:
We may eventually want to replace this with a string literal union type. There are benefits and tradeoffs to enums. Mostly tradeoffs in my opinion.

No need to change anything on this PR, but let's chat about it later.